### PR TITLE
feat: husky pre commit message, lint and format. Also @ aliases for importing

### PR DESCRIPTION
**Pull request by fgomensoro**

### CODE REVIEW

File: src/components/LoginTest/LoginTest.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function LoginPrueba() {
```
Comment (elios11) [inline src/components/LoginTest/LoginTest.jsx:1]: What's this for?

**Comment by fgomensoro** [comment]
Commits without the formatting were for testing precommit message validation.
When they are squashed and merged they will not be taken into account.

**Review by elios11 — APPROVED** [review]
looks ok!

File: src/components/LoginTest/LoginTest.jsx (hunk 1 above)
Comment (fgomensoro) [reply src/components/LoginTest/LoginTest.jsx:1]: A simple component connected to login page

**Review by valselviz — APPROVED** [review]

**Pull request merged**